environment.yml

```diff
@@ -1,247 +1,22 @@
 name: nf1_analysis
 channels:
-  - defaults
+  - conda-forge
 dependencies:
```

Comment:
if you have any notebooks in this repo I recommend adding ipython to the environment file for reproducibility

Reply:
Sounds good, added!